INSTALL.txt

```diff
@@ -61,16 +60,32 @@ workspace_base_dir = /home/workspace_base
 workspace_storage_base_dir = /home/workspace_storage_base
 workspace_cache_base_dir = /home/workspace_cache
 
-The XT_RCAR_EVAPROPRIETARY_DIR, XT_ANDROID_PREBUILDS_DIR variables under [local_conf] section depends
-on whether prebuilt EVA proprietary packages are going to be used.
+3.3 Please, change the variables under [local_conf] section:
+
+The XT_GUESTS_INSTALL and XT_GUESTS_BUILD variables select the guest domain
+to be built and installed.
+
+For example (DomU),
+
+XT_GUESTS_INSTALL = "domu"
+XT_GUESTS_BUILD = "domu"
+
+For example (DomA),
+
+XT_GUESTS_INSTALL = "doma"
+XT_GUESTS_BUILD = "doma"
+
+The XT_RCAR_EVAPROPRIETARY_DIR (DomD and DomU), XT_ANDROID_PREBUILDS_DIR (DomA) variables
+depend on whether prebuilt EVA proprietary packages are going to be used.
+
```

Comment:
This two lines duplicate following line "If you are going to use prebuilt EVA proprietary packages,.."

Reply:
Will rephrase